aiohttp/connector.py

```diff
@@ -181,7 +181,11 @@ def __init__(self, *, keepalive_timeout=sentinel,
         self._acquired_per_host = defaultdict(set)
         self._keepalive_timeout = keepalive_timeout
         self._force_close = force_close
-        self._waiters = defaultdict(list)
+
+        # {host_key: FIFO list of waiters}
+        # NOTE: this is not a true FIFO because the true order is lost amongst
+        #       the dictionary keys
```

Comment:
I guess that this can be removed, the FIFO is for each dictionary key. Different keys mean different tuple values of `(hosts, port)` which FIFO does not make sense.

Reply:
each deque is indeed a FIFO as the first one in will be the first one out, however across keys it's not a FIFO because it currently iterates across keys (which theoretically can be in any order) when choosing from which FIFO to release.